fix the background color of the sidebar on 'medium' sized devices

ingshtrom opened the pull request:
<!--Thanks for your contribution. See [CONTRIBUTING](CONTRIBUTING.md)
    for this project's contribution guidelines. Remove these comments
    as you go.

    DO NOT edit files and directories listed in _data/not_edited_here.yaml.
    These are maintained in upstream repos and changes here will be lost.

    Help us merge your changes more quickly by adding details and setting metadata
    (such as labels, milestones, and reviewers) over at the right-hand side.-->

### Proposed changes
This fixes #11366 by adding the background color for the correct class name. It
seems that `.sidebar-home` used to be used a lot, but I couldn't find its usage
anywhere (when loading up the docs app), so I stuck to using `.sidebar`, which
seemed to work :smile:


### Unreleased project version (optional)

<!--If this change only applies to an unreleased version of a project, note
    that here and base your work on the `vnext-` branch for your project. If
    this doesn't apply to this PR, you can remove this whole section.
    Set a milestone if appropriate. -->

### Related issues (optional)

Fixes #11366

<!--Refer to related PRs or issues: #1234, or 'Fixes #1234' or 'Closes #1234'.
    Or link to full URLs to issues or pull requests in other Github projects -->

usha-mandya commented:
Thanks @ingshtrom. Looks like the issue still exists

![image](https://user-images.githubusercontent.com/47779042/93362740-435c3a80-f83e-11ea-9317-05493e8f3e3b.png)

ingshtrom commented:
Oops! I fixed this, and then noticed that the header and side-bar changes color (black navbar, and darker sidebar) at the lowest resolutions. I fixed that, but maybe it was purposeful. Please let me know if it was and I can roll that back.

usha-mandya commented:
Thank you @ingshtrom. It looks fine to me. @StefanScherer Do you mind taking another look when you get a chance? Thanks.

StefanScherer commented on _scss/_mobile.scss:
```diff
@@ -117,16 +118,10 @@
     }
-    .nav-secondary-tabs,
-    .nav-secondary {
-        background-color: $black;
```
Looks good so far, only thing I see is that on mobile the top bar is blue in light mode, but when I rotate my mobile to landscape it turns black. 
Previously on mobile it always was black. I think this change here was for black -> blue.
I like the blue background better in light mode.
But we should also fix it for landscape mode further down line 234 ff.

ingshtrom replied:
Right, what I noticed was that on mobile the navbar (top bar) was black so I was trying to make it consistent with non-mobile views. I will change this for landscape mode as well. Nice find!

ingshtrom commented:
Oh, man.... I also noticed the search input changes based on the size of the screen... I'll fix that too.

ingshtrom commented:
PTAL. I think I got it this time. Mobile portrait and landscape are looking good. Just making a desktop screen smaller is looking good.

StefanScherer reviewed (approved):
LGTM 

Thanks Alex! Works fine now on mobile (portrait and landscape).

usha-mandya reviewed (approved):
LGTM. Thank you Alex.
